File: Q5/Q5.py
# Check if a string is a palindrome.

def checkPalindrome(s: str):
    checkStr = s[::-1]
    if s == checkStr:
        print(f"{s} is a palindrome")
    else:
        print(f"{s} is not a palindrome")

checkPalindrome("sister")
checkPalindrome("madam")

File: Q5/Q5.ts
// Check if a string is a palindrome.

function isPalindrome(str: String) {
  const checkStr = str.split("").reverse().join("");
  if (str == checkStr) {
    console.log(`${str} is palindrome`);
  } else {
    console.log(`${str} is not palindrome`);
  }
}

isPalindrome("zizi");
isPalindrome("azizi");
